clang/docs/ReleaseNotes.rst

```diff
@@ -843,6 +843,9 @@ Bug Fixes to C++ Support
 - Fix crash when parsing nested requirement. Fixes:
   (`#73112 <https://github.com/llvm/llvm-project/issues/73112>`_)
 
+- Fixed a crash caused by using return type requirement in a lambda. Fixes:
+  (`#64607 <https://github.com/llvm/llvm-project/issues/64607>`_)
```

Comment:
Please list all the bugs it fixes.

Reply:
Thanks for review!
I have added the list of related bugs and improved the comments.